*** Settings ***
Test Setup        Run Keywords    fs_loadDataFromFileNewVersion    ${CURDIR}/Data.xlsx    AND    Open Application    ${v_remoteUrl}    ${v_alias}
Test Teardown     Close All Applications
Resource          ../../Resources/Android/SignInKeywords.txt
Resource          ../../Resources/Android/StringAFile.txt

*** Test Cases ***
String File Successful

test
    ${v_isConnectedToDatabase}=    Run Keyword And Ignore Error    DatabaseLibrary.Connect To Database Using Custom Params    adodbapi    'Provider=SQLOLEDB.1;Data Source=${v_databaseSource};Initial Catalog=${v_databaseName};User Id=${v_databaseUsername};Password=${v_databaseUserPassword};'
    Run Keyword If    '${v_isConnectedToDatabase[0]}'=='FAIL'    Fail    Could not connect to database, the error was: "${v_isConnectedToDatabase[1]}"
    @{v_queryResult}=    DatabaseLibrary.Query    select * from UserEmailAddress where EmailAddress='staandroidsecondunvery@yopmail.com'
    Log Many    @{v_queryResult}

fs[4735]_Where can String_From Menu ( Plus "+" on top corner right)
    Log    1234

fs[]_Share file using FileString from_File manager when App has logn In already

fs[]_Share file using FileString from_File manager when App has not Sign In yet
